### PR TITLE
Added save and restore tutorial

### PR DESCRIPTION
**Pull request by hunkim**
This can be very useful for incremental learning, but the explanation in the TensorFlow.org seems not enough.

### CODE REVIEW

File: 10_save_restore_net.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import numpy as np
+import input_data
+import os
+
+'''
+This shows how to save/restore your model (trained variables).
+To see how it works, please stop this program during training and resart.
+
+This network is the same as 3_net.py
+'''
```
Comment (hunkim) [inline 10_save_restore_net.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/hunkim/TensorFlow-Tutorials/file/2429996717/issues/source?bid=3237149&fileBranchId=3241332#l13)

**Comment by nlintz** [comment]
Hey Sung,
I'm seeing a codacy error in the checks. Do you think we should go ahead and merge anyways or do you want to modify the PR to meet codacy requirements?

**Comment by hunkim** [comment]
@nlintz They are mostly style checking, but let me see if I can remove.

File: 10_save_restore_net.py (hunk 1 above)
Comment (hunkim) [inline 10_save_restore_net.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/hunkim/TensorFlow-Tutorials/file/2433005969/issues/source?bid=3237149&fileBranchId=3241332#l13)

Comment (hunkim) [inline 10_save_restore_net.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/hunkim/TensorFlow-Tutorials/file/2433005969/issues/source?bid=3237149&fileBranchId=3241332#l13)

Comment (hunkim) [inline 10_save_restore_net.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/hunkim/TensorFlow-Tutorials/file/2433022649/issues/source?bid=3237149&fileBranchId=3241332#l13)

Comment (hunkim) [inline 10_save_restore_net.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/hunkim/TensorFlow-Tutorials/file/2433022649/issues/source?bid=3237149&fileBranchId=3241332#l13)

Comment (hunkim) [inline 10_save_restore_net.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/hunkim/TensorFlow-Tutorials/file/2433079257/issues/source?bid=3237149&fileBranchId=3241332#l13)

Comment (hunkim) [inline 10_save_restore_net.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/hunkim/TensorFlow-Tutorials/file/2433079257/issues/source?bid=3237149&fileBranchId=3241332#l13)

Comment (hunkim) [inline 10_save_restore_net.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/hunkim/TensorFlow-Tutorials/file/2433098389/issues/source?bid=3237149&fileBranchId=3241332#l13)

Comment (hunkim) [inline 10_save_restore_net.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/hunkim/TensorFlow-Tutorials/file/2433098389/issues/source?bid=3237149&fileBranchId=3241332#l13)

**Comment by hunkim** [comment]
@nlintz It seems I don't see codacy checks here. Perhaps, it won't pull-up updates in PR. However, it seems OK now.

One more thing. Should we go with two digit numbers, 00, 01 to order files such as 10_ properly?

image

**Comment by nlintz** [comment]
looks good, merged

**Pull request merged**